settings.lua

```diff
@@ -80,6 +80,13 @@ data:extend({{
     default_value = 30,
     minimum_value = 0,
     maximum_value = 3600
+}, {
+    type = "int-setting",
+    name = "logistics_fullfillment_timeout",
+    setting_type = "runtime-global",
+    default_value = -1,

```

Comment:
maybe having 0 be the disabled state would be a better fit (timeout of 0 ticks would not be really useful anyways) and would make the check in the code cleaner because its not inherently clear why there is a check if the global value is not `-60`.

Reply:
yeah sure, im not particular on that really.